lektor/i18n.py

```diff
@@ -13,7 +14,8 @@
 
 translations = {}
 for _lang in KNOWN_LANGUAGES:
-    with open(os.path.join(translations_path, _lang + '.json')) as f:
+    with open(os.path.join(translations_path, _lang + '.json'),
+              encoding="utf8") as f:
```

Comment:
@rlaverde I advice using `io.open(` instead of overloading "normal" `open` on python2, it makes the code easier to follow and understand, even if it is the only `open` in the file